tests/test_formats.py

```diff
@@ -364,11 +372,23 @@ def test_from_tree_sequence_bad_populations(self):
 
     def test_from_tree_sequence_simple(self):
         ts = tsutil.get_example_ts(10)
-        sd1 = formats.SampleData(sequence_length=ts.sequence_length)
+        sd1 = formats.SampleData(
```

Comment:
Why change all these tests? It doesn't look like ``verify_data_round_trip`` checks time_units, so why alter these tests? We want to know that old code will continue to work, so *not* changing existing tests would be much more reassuring.